tools/src/commands/AndroidNativeUnitTests.ts

```diff
@@ -88,7 +90,9 @@ export async function androidNativeUnitTests({
 
   const [androidPackagesTestedUsingBareProject, androidPackagesTestedUsingExpoProject] = partition(
     androidPackages,
-    (element) => packagesNeedToBeTestedUsingBareExpo.includes(element.packageName)
+    // (element) => packagesNeedToBeTestedUsingBareExpo.includes(element.packageName)
+    // Test all packages using bare expo
+    (element) => true
```

Comment:
i think that's temporary. would be good to clean up that file before merging the pr.

Reply:
@Kudo thanks for the tips. I think it might be good to either use `apps/native-tests` for this, or create a new app specifically for Android unit/instrumentation tests.

Reply:
@Kudo it seems like `native-tests` will not work. I opted to use `bare-expo`, and modify its `package.json` to autolink `expo-updates` before running tests, and restore the file at the end. `expo-updates` instrumented tests are broken at the moment, so they are stubbed out and will be fixed separately. @wschurman